### DISCUSSION TITLE
38 课的程序跑的时候报错

### DISCUSSION THREAD

**Discussion by mpyexiu** [AmazingAng/WTF-Solidity#475] [General]
<img width="1007" alt="截屏2023-04-10 17 31 04" src="https://user-images.githubusercontent.com/130408097/231002948-e9f2d6da-d4f7-4449-8df8-e37400c6fea6.png">
在跑list的时候遇到了这样的问题，有朋友知道怎么解决么，初学者实在看不出问题在哪里

**Comment by Azleal** [comment]
你是不是在调用函数的时候传入value了, 但这个函数不是`payable`? 只有这个看不出是什么问题

**Comment by szlishun54** [comment]
我也啥都没看，据我经验推测，你可看看solidyt版本是否对？
